model-client/build.gradle.kts

```diff
@@ -69,27 +69,22 @@ kotlin {
             dependencies {
                 kotlin("stdlib-jdk8")
 
-                implementation("io.vavr:vavr:0.10.3")
-                implementation("org.apache.commons:commons-lang3:3.11")
-                implementation("com.google.guava:guava:29.0-jre")
-                implementation("org.glassfish.jersey.core:jersey-client:2.31")
-                implementation("org.glassfish.jersey.inject:jersey-hk2:2.31")
-                implementation("org.glassfish.jersey.media:jersey-media-sse:2.31")
-                implementation("javax.xml.bind:jaxb-api:2.3.1")
```

Comment:
```
implementation("org.glassfish.jersey.core:jersey-client:2.31")
implementation("org.glassfish.jersey.inject:jersey-hk2:2.31")
implementation("org.glassfish.jersey.media:jersey-media-sse:2.31")
implementation("javax.xml.bind:jaxb-api:2.3.1")
```
Is it intentional to remove these entirely?

Reply:
These are old unused deps. The green build confirms that.